lib/i18n/js/engine.rb

```diff
@@ -9,56 +10,58 @@ class SprocketsExtension
     end
 
     class Engine < ::Rails::Engine
-      if JS::Dependencies.sprockets_supports_register_preprocessor?
```

Comment:
Oh yeah, now I see. Btw even if sprockets is here, blank rails 3 app doesn't have "sprockets/version" required. I'll craft a workaround